src/OrbitDB.js

```diff
@@ -345,6 +345,11 @@ class OrbitDB {
     return Object.keys(databaseTypes).includes(type)
   }
 
+  static addType (type, store) {
+    if (databaseTypes[type]) throw new Error(`Type already exists: ${type}`)
+    databaseTypes[type] = store
```

Comment:
`databaseTypes` is defined on https://github.com/garbados/orbit-db/blob/1ef24dfa4be6884103c72e3762f163b8ae29a7a3/src/OrbitDB.js#L21 as `const`. Can you change that to be `let` to make sure the type describes it's a variable that will/can change?